### ISSUE TITLE
Cannot get tht same accuracy.

### ISSUE THREAD

**Issue by iimmortall** [vincenzodentamaro/aucoresnet#3]
Nice Work,  I use your code and Coswara-Data, but I just get 0.63 AUC with finetuning parameters. I check the training log, and I found the model overfit. Is your code complete?

**Comment by vincenzodentamaro** [comment]
The code is complete. Dataset were not published for copyright concerns. Please describe what you used for train and test. Also pay attention to sampling frequency in the datasets.
More information are needed.

**Comment by iimmortall** [comment]
I use the cough audio data from the Coswara-Data and divide the dataset into train, validation, and test. And the parameters are the same as your code, sr 22050, max_length 10, etc. 
And I also confuse about your results on virufy-data, validation's  AUC is 0.5, but the test AUC is 1.0,  and the validation data are the same as the test data.

**Comment by iimmortall** [comment]
I use the cough audio data from the Coswara-Data and divide the dataset into train, validation, and test. And the parameters are the same as your code, sr 22050, max_length 10, etc. 
And I also confuse about your results on virufy-data, validation's  AUC is 0.5, but the test AUC is 1.0,  and the validation data are the same as the test data.

**Comment by vincenzodentamaro** [comment]
Just a few points:

1. When you do k-fold cross validation (usually when data is not very big) the validation set _IS_ the test set. In our case we are doing a particular case of K-Fold Cross Validation called K-Fold interpatient separation scheme, which is used to make sure that no data from the same person falls in both train and test sets and the dataset is balanced per class and per people.
2. Final results are computed per person and not per audio 
3. I don't remember the Coswara dataset sampling frequency, but make sure you downsample to 22khz before passing the audio to the AUCO
4. The Virufy dataset used is a toy dataset very very small so take its result as a grain of salt. In addition,  pay attention, results shwn during training by tensorflow are per-instance, while final results are computer per person.

**Comment by iimmortall** [comment]
Thank you for you suggestion, I have got the correct result.
